### DISCUSSION TITLE
Welcome to Elsa Workflows Discussions!

### DISCUSSION THREAD

**Discussion by sfmskywalker** [elsa-workflows/elsa-core#501] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PeterLaComb-Availity** [comment]
My name is Peter LaComb, and I'm a software engineer for [Availity](http://availity.com/). We have a couple of teams that are using Elsa for tracking the flow of EDI file validation, transformation, and delivery. 

I intend to spend some time focusing on adding some meaningful unit tests to Elsa. Part of my desire there is to flesh out areas that have been difficult or where our learning curve has been steep. I believe that tests which cover things that ought to work, or that don't work the way we thought thought they would, will lead to refined documentation, feature improvement suggestions, and potentially pull-requests to add or update those features.

**Comment by viswajwalith** [comment]
Hi Mate, Even tough it is open source, is there any licensing  cost involved when we make use of this in our commercial product development and distribute this to other customers?

**Reply by sfmskywalker** [reply]
No costs at all - you are free to use Elsa in any way you like, including using & redistributing it in your own commercial product.

**Reply by viswajwalith** [reply]
Thanks for the Quick Response 👍
